### ISSUE TITLE
About pre-training

### ISSUE THREAD

**Issue by ChengJiacheng** [Sha-Lab/FEAT#5]
Hi Fusheng,

We have released the pre-trained model's weight [here](https://drive.google.com/open?id=14Jn1t9JxH-CxjfWy4JmVpCxkC9cDqqfE). Our pretraining procedure is basically the same as [this](https://arxiv.org/abs/1807.05960), trained on 64 training classes and validated on 16 validation classes. Model selection is based on nearest neighbor classifier's result (accuracy) on 16 validation classes. Please refer to [this](https://github.com/pytorch/examples/tree/master/imagenet) repo for your reference of implementation details.

_Originally posted by @hexiang-hu in https://github.com/Sha-Lab/FEAT/issues/1#issuecomment-450611114_

Hi hexiang,

I am planning to implement the pretraining by myself.
The repo you list seems not applicable for training a wide residual net used in the paper
Hence, I am not sure if I understand the pretraining procedure correctly.
Is it true that I just need to add a 640*64 linear layer after the embedding (model.endoder) and then train the 64-way classifier under the cross-entropy loss?

Any information will be greatly appreciated.
Thanks!
